ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3618,10 +3618,10 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-shared-to-local-gateway-mode-migration
-- as: vsphere-ipi-ovn-shared-to-local-gw-migration-f28-destructive
-  cron: 36 6 6 * *
+- as: vsphere-ipi-ovn-winc-p2-f14
+  cron: 32 12 8,23 * *
   steps:
-    cluster_profile: vsphere-2
+    cluster_profile: vsphere-connected
```

Comment:
The cluster should be vsphere-2 or vsphere-connected-2. You can have a look at the current [config file](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml#L3673) for reference